File: duplicate-assert/csharp-xunit/FileLengthCheckTest_testExtensions.cs
using Xunit;

public class FileLengthCheckTest : AbstractModuleTestSupport
{
    [Fact]
    public void TestExtensions()
    {
        var check = new FileLengthCheck();
        check.SetFileExtensions("java");
        Assert.Equal(".java", check.GetFileExtensions()[0], "extension should be the same");
        check.SetFileExtensions(".java");
        Assert.Equal(".java", check.GetFileExtensions()[0], "extension should be the same");

        var exception = Assert.Throws<ArgumentException>(() => check.SetFileExtensions(null));
        Assert.Equal("Extensions array can not be null", exception.Message, "Invalid exception message");
    }
}

File: smelly/duplicate-assert/java-junit/FileLengthCheckTest_testExtensions.java
import org.junit.jupiter.api.Test;
import static org.junit.jupiter.api.Assertions.*;

public class FileLengthCheckTest extends AbstractModuleTestSupport {

    @Test
    public void TestExtensions() {
        FileLengthCheck check = new FileLengthCheck();
        check.setFileExtensions("java");
        assertEquals(".java", check.getFileExtensions()[0], "extension should be the same");
        check.setFileExtensions(".java");
        assertEquals(".java", check.getFileExtensions()[0], "extension should be the same");

        Exception exception = assertThrows(IllegalArgumentException.class, () -> check.setFileExtensions(null));
        assertEquals("Extensions array can not be null", exception.getMessage(), "Invalid exception message");
    }
}
